Dockerfile

```diff
@@ -118,7 +118,7 @@ RUN ${APPLIANCE_ROOT}/setup && \
 WORKDIR ${APP_ROOT}
 RUN source /etc/default/evm && \
     export RAILS_USE_MEMORY_STORE="true" && \
-    npm install gulp bower -g && \
+    npm install gulp bower yarn -g && \
```

Comment:
Don't need bower anymore for the SUI (Classic UI might still need it, but it should go under a section for the classic UI)